modules/machine-about-phases.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * machine_management/machine-phases-lifecycle.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="machine-about-phases_{context}"]
+= Machine phases
+
+As a machine moves through its lifecycle, it passes through different phases. Each phase is a basic representation of the state of the machine.
+
+`Provisioning`:: There is a request to provision a new machine. The machine does not yet exist and does not have an instance, a provider ID, or an address.
+
+`Provisioned`:: The machine exists and has a provider ID or an address. The cloud provider has created an instance for the machine. The machine has not yet become a node and the `status.nodeRef` section of the machine object is not yet populated.
+
+`Running`:: The machine exists and has a provider ID or address. Ignition has run successfully and the cluster machine approver has approved a certificate signing request (CSR). The machine has become a node and `status.nodeRef` section of the machine object contains node details.
+
+`Deleting`:: There is a request to delete the machine. The machine object has a `DeletionTimestamp` field showing the time the deletion request occurred.
+
+`Failed`:: There is an unrecoverable problem with the machine. The following are some possible causes:
++
+--
+* If the configuration for the requested machine is invalid, creation fails with an `invalidConfigurationMachineError` error message.
+* If the machine does not exist but has an associated provider ID or address, creation fails.
```

Comment:
Not only true for creation. If the instance goes away while the Machine state is Running, eg cloud provider deletes it outside of MAPI, then the Machine will go failed